testutil/server/configurator/configurator.go

```diff
@@ -81,7 +81,7 @@ func (c Fixture) Setup() server.Fixture {
 var _ server.Fixture = Fixture{}
 
 // TxConn implements the Fixture.TxConn method
-func (c Fixture) TxConn() grpc.ClientConnInterface {
+func (c Fixture) TxConn(sdk.AccAddress) grpc.ClientConnInterface {
```

Comment:
why do add `sdk.Address` param here? could that be used to define a module or sub module account?

Reply:
Yes, I think a TxConn should always be in reference to a specific account. Otherwise, how do we specify who does the signing?

Reply:
ended up reverting this